manager-bundle/bin/contao-console

```diff
@@ -28,8 +28,7 @@ if (file_exists(__DIR__.'/../autoload.php')) {
     $projectDir = \dirname(__DIR__, 4);
 }
 
-/** @var Composer\Autoload\ClassLoader $loader */
-$loader = require $projectDir.'/vendor/autoload.php';
+require $projectDir.'/vendor/autoload.php';
```

Comment:
this seems unrelated to the PR and should not be changed

Reply:
It is a left-over from Symfony 3. @m-vo and I have discussed removing this so the code is the same in all binaries.